toxcore/group_chats.c

```diff
@@ -156,6 +156,11 @@ typedef enum Group_Sync_Flags {
     GF_STATE      = (1 << 2), // 4
 } Group_Sync_Flags;
 
+typedef enum Group_CustomPkts_Direction {
+    GC_CUSTOMPKTS_SENDING      = (1 << 0), // 1
```

Comment:
Why are we using bitmasks? There will only ever be two possibilities and never at the same time. I don't mind an enum but the values shouldn't be bit set.